packages/@aws-cdk/aws-cognito/test/user-pool-client.test.ts

```diff
@@ -540,4 +540,64 @@ describe('User Pool Client', () => {
       },
     })).toThrow(/disableOAuth is set/);
   });
+
+  test('token validity', () => {
+    // GIVEN
+    const stack = new Stack();
+    const pool = new UserPool(stack, 'Pool');
+
+    // WHEN
+    pool.addClient('Client1', {
+      userPoolClientName: 'Client1',
+      accessTokenValidity: Duration.minutes(60),
+      idTokenValidity: Duration.minutes(60),
+      refreshTokenValidity: Duration.days(30),
+    });
+    pool.addClient('Client2', {
+      userPoolClientName: 'Client2',
+      accessTokenValidity: Duration.minutes(60),
+    });
+    pool.addClient('Client3', {
+      userPoolClientName: 'Client3',
+      idTokenValidity: Duration.minutes(60),
+    });
+    pool.addClient('Client4', {
+      userPoolClientName: 'Client4',
+      refreshTokenValidity: Duration.days(30),
+    });
```

Comment:
can you add another client with none of the token validity specified that asserts that the TokenValidityUnits is absent?

```ts
expect(stack).toHaveResource('AWS::Cognito::UserPoolClient', {
  ClientName: ...,
  TokenValidityUnits: ABSENT,
  IdTokenValidity: ABSENT,
  RefreshTokenValidity: ABSENT,
  AccessTokenValidity: ABSENT,
});
```